plugins/uefi/fu-plugin-uefi.c

```diff
@@ -403,6 +428,10 @@ fu_plugin_update (FuPlugin *plugin,
 	str = _("Installing firmware update…");
 	g_assert (str != NULL);
 
+	/* make sure that the ESP is mounted */
+	if (!fu_plugin_uefi_esp_mounted(plugin, error))
```

Comment:
space before the bracket please. Sorry to nitpick.

Reply:
One of these days i'll find a plugin for my editor that corrects this automatically because i'll never get it right :)

Reply:
Oh wait, it looks like they might have it built in! https://github.com/Microsoft/vscode/issues/15386